tajo-core/src/test/java/org/apache/tajo/engine/planner/physical/TestExternalSortExec.java

```diff
@@ -61,7 +61,7 @@
   private LogicalPlanner planner;
   private Path testDir;
 
-  private final int numTuple = 3000000;
```

Comment:
This is fixed for proper test of ExternalSortExec by request (It's not using merge sort but memory scanner). I'll fix that in a following issue.